widgets/AttributesTable/_QueryTaskMixin.js

```diff
@@ -60,8 +60,8 @@ define([
                 outStatistics: null,
                 pixelSize: null,
                 relationParam: null,
-                spatialRelationship: Query.SPATIAL_REL_INTERSECTS
-
+                spatialRelationship: Query.SPATIAL_REL_INTERSECTS,
+                includeLayerDefinitions: false
```

Comment:
this should be singular `includeLayerDefinition` since we are technically only using a single layer definition - either from the feature layer or from the one sub layer of a dynamic layer.